charts/prometheus-thanos/templates/ruler-statefulset.yaml

```diff
@@ -82,11 +83,41 @@ spec:
           volumeMounts:
             - mountPath: /etc/thanos-ruler
               name: config
+            - mountPath: /etc/thanos-ruler/external
+              name: external-config-volume
             - mountPath: /data
               name: storage-volume
           {{- with .Values.ruler.volumeMounts }}
             {{- toYaml . | nindent 12 }}
           {{- end }}
+{{- if .Values.ruler.sidecar.enabled }}
```

Comment:
I'd like to see a documentation in the README for the whole rule import mechanism.

Reply:
I updated the README